client/my-sites/domains/domain-management/dns/dns-records.jsx

```diff
@@ -207,6 +214,10 @@ class DnsRecords extends Component {
 			return null;
 		}
 
+		const targetPlatform = selectedDomain?.isGravatarDomain
+			? translate( 'Gravatar profile' )
+			: translate( 'WordPress.com site' );
```

Comment:
I know we're just replacing 2 words, but I think it might be safer (from an i18n perspective) to repeat the whole sentence here. See i18n best practices, particularly the "It’s ok to “almost” repeat yourself" section (PCYsg-3hI-p2).

Something like:

```suggestion
const message = selectedDomain?.isGravatarDomain ?
	translate(
		'Your domain is not using default A records. This means it may not be pointing to your Gravatar profile correctly. To restore default A records, click on the three dots menu and select "Restore default A records". {{defaultRecordsLink}}Learn more{{/defaultRecordsLink}}.'
	)
	: translate(
		'Your domain is not using default A records. This means it may not be pointing to your WordPress.com site correctly. To restore default A records, click on the three dots menu and select "Restore default A records". {{defaultRecordsLink}}Learn more{{/defaultRecordsLink}}.'
	)
```

Reply:
Ah I was wondering about that, my first commit had a this approach. Thanks for the tip, I'll swap the approach :) 

https://github.com/Automattic/wp-calypso/pull/99384/commits/c0aeea574e0369880035ebc9b4b7895cf41b728f#diff-0d221efe0f85d39c773de639d803ef8bc82af5297d23c57e3026a4a1af0a41d7R217

Reply:
ah, jinx... I just left a similar comment before seeing this. :D

Reply:
Thanks for that reminder, @leonardost! Makes sense.